### PR TITLE
Upgrades

### PR DESCRIPTION
**Pull request by pvinis**
# Summary

## Checklist

- [ ] I tested this thoroughly
- [ ] I added the documentation in `README.md` (if needed)

### CODE REVIEW

File: src/components/common/DiffViewer.js [hunk 1]
```diff
@@ -69,9 +69,8 @@ const DiffViewer = ({
     const fetchDiff = async () => {
       setLoading(true)
 
-      const response = await (await fetch(
-        getDiffPatchURL({ fromVersion, toVersion })
-      )).text()
+      let response = await fetch(getDiffPatchURL({ fromVersion, toVersion }))
+      response = response.text()
```
Comment (lucasbento) [inline src/components/common/DiffViewer.js]: Why did you need to do this?

Comment (pvinis) [reply src/components/common/DiffViewer.js]: I actually changed it to simplify the nestesed awaits, and I forgot to add the await on the second line. this is reverted on master by the fix PR from Nat.
